core/src/hpke.rs

```diff
@@ -261,7 +261,7 @@ impl HpkeKeypair {
     }
 }
 
-/// HPKE configuration compatible with the output of `divviup hpke-config generate`.
+/// HPKE configuration compatible with the output of `divviup collector-credential generate`.
 #[derive(Debug, Clone, PartialEq, Eq, Deserialize)]
 pub struct DivviUpHpkeConfig {
```

Comment:
I'm pretty sure the answer is "no", but I just want to check: is there a crate shared across Janus and `divviup-api` where type definitions like this could live?

Reply:
I think janus_core is the closest there is, although I don't think divviup-api relies on it for its definition of the HPKE config (seeing as divviup-api provides a `token` field).

This topic has also come up recently wrt sharing tracing/metrics subscribers. https://isrg.slack.com/archives/C0167LT4C73/p1698861551769459. I like the idea but I see it as a nice-to-have cleanup.